applications/app/views/fragments/gallerySlot.scala.html

```diff
@@ -1,8 +1,16 @@
 @(
     id: Int,
     isMobile: Boolean = false
-)
+)(implicit request: RequestHeader)
 @import conf.switches.Switches.CommercialSwitch
+@import mvt.CommercialGalleryBannerAds
+
+@additionalSizes = @{
+    if(CommercialGalleryBannerAds.isParticipating)
+        Seq("970,250", "970,350")
+    else
+        Seq.empty
+}
```

Comment:
This template has already a bunch of logic. Could it leave outside of the template code where is harder to read and test?